### PR TITLE
[INT2-110]  Add using yearn

### PR DESCRIPTION
**Pull request by CRIPU**
thanks!

### CODE REVIEW

File: using-yearn.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# yearnの使い方
```
Comment (DarkGhost7) [inline using-yearn.md:1]: > yearnの使い方
change to `Yearnの使い方`

File: using-yearn.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+こちらが使い方です：
+
+まず、右上のボタンで**ウォレットの接続**を行います。複数の種類のウォレットがサポートされていますが、ほとんどの人は、Chromeの拡張機能として、またはAppleやAndroidのアプリストアを通じて無料でダウンロードできる[MetaMask](https://metamask.io/)を使用しています。ウォレットがイーサリアムのネットワークに接続されていることを確認してください。
+
+<p align="center">
+  <img width="1266.75" height="345.75" src="https://i.imgur.com/H0Uc8e8.png">
+</p>
+
+
```
Comment (DarkGhost7) [inline using-yearn.md:13]: delete line 13 extra space

File: using-yearn.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+<p align="center">
+  <img width="914.26" height="360.75" src="https://i.imgur.com/LGdRAMQ.png">
+</p>
+
+3\. Approve前の場合は「Approve」を、Approve済の場合は「入金」ボタンをクリックしてください。
+
+4\. あなたのウォレットが取引の確認を求めます。これには約3分かかりますが、「ネットワークに高いガス料金を支払う」(https://blog.leverj.io/how-to-set-the-gas-limit-and-gas-price-in-metamask-1b33c38c32fd)ことで早くすることができます。トランザクションが止まってしまった場合は、[ガイド](https://metamask.zendesk.com/hc/en-us/articles/360015489251-How-to-Speed-Up-or-Cancel-a-Pending-Transaction)を参照して、トランザクションのスピードアップまたはキャンセルを行ってください。
+
+
```
Comment (DarkGhost7) [inline using-yearn.md:28]: delete extra space at line 28

File: using-yearn.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+  <img width="910.5" height="361.5" src="https://i.imgur.com/Y3NsCRb.png">
+</p>
+
+3\. 「Withdraw」をクリックします。
+
+4\. ウォレットが取引の確認を求めてきます。詳細は上記のステップ4を参照してください。
+
+5\. トランザクションが成功すると、トークンは再びあなたのウォレットに表示されます。
+
+
```
Comment (DarkGhost7) [inline using-yearn.md:56]: delete extra space at line 56

File: using-yearn.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+</p>
+
+3\. 出金時にどの資産を受け取りたいかを選択します。選択肢は、crvSTETH、ETH、BTC、DAI、USDC、USDTとなります。
+
+4\. 出金したい金額を入力するか、「Max」をクリックして残高全体を引き出すことができます。
+
+5\. 必要に応じて approve を確認し、出金取引を承認してください。
+
+6\. 取引が成功すると、トークンがあなたのウォレットに再び表示されます。
+
```
Comment (DarkGhost7) [inline using-yearn.md:100]: extra space line 100

File: using-yearn.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+## 資金をチェックするツール 
+
+あなたの資産がvaultに入っている間、米ドルの残高がどのように変化するかを確認したい場合は、あなたのウォレットを[zapper.fi](https://zapper.fi)または同様のポートフォリオ・トラッキングプロダクトに接続してください。これは、Vaultがお客様にどれだけの利益をもたらしたかを知る最も簡単な方法でもあります。
+
+お客様の残高が継続的に増えることはありません。利益は harvest() 関数が呼ばれたときに、あなたの金庫のシェアに分配されますが、これは変動的に起こります。
+
```
Comment (DarkGhost7) [inline using-yearn.md:107]: extra space 107

**Review by DarkGhost7 — CHANGES_REQUESTED** [review]

**Comment by CRIPU** [comment]
fixed it ! thank you so much

**Review by DarkGhost7 — APPROVED** [review]
lgtm

**Pull request merged**